packages/hs-react-ui/src/components/Table/Table.tsx

```diff
@@ -151,31 +194,76 @@ const SortIcon = styled(Icon)`
   `}
 `;
 
+/** Start of variables */
+
+const defaultCollapsed: collapsedState = {};
+
+// Default expansion column added
+const collapsexpandedIconColumn = {
```

Comment:
I think we're missing an `e` here lol
```suggestion
const collapseExpandedIconColumn = {
```